fastai/datasets.py

```diff
@@ -132,6 +132,7 @@ class Config():
     DEFAULT_CONFIG_PATH = DEFAULT_CONFIG_LOCATION + '/config.yml'
     DEFAULT_CONFIG = {
         'data_path': DEFAULT_CONFIG_LOCATION + '/data',
+        'data_archive_path': DEFAULT_CONFIG_LOCATION + '/data/archive',
```

Comment:
I thought that we agreed that the default should remain as it was. And letting the user to override it. This is a different implementation.

Reply:
OK. I'll change this to fit the specs.